src/User.php

```diff
@@ -3271,6 +3271,8 @@ public function showMyForm($target, $ID)
                     echo "<td colspan='2'></td>";
                 }
                 echo "</tr>";
+            } else {
+                echo "<tr class='tab_bg_1'><td colspan='2'></td></tr>";
```

Comment:
Result seems to be different depending on the following condition : `if ($extauth|| !Session::haveRight("password_update", "1")) {`. Indeed the code from line 3267 to 3272 seems to indicate that it may require an additional placeholder.

Could you check ?